examples_linux/interrupt_configure.py

```diff
@@ -97,14 +105,25 @@
 def interrupt_handler():
     """This function is called when IRQ pin is detected active LOW"""
     print("\tIRQ pin went active LOW.")
-    tx_ds, tx_df, rx_dr = radio.whatHappened()  # update IRQ status flags
-    print(f"\ttx_ds: {tx_ds}, tx_df: {tx_df}, rx_dr: {rx_dr}")
+    flags = radio.clearStatusFlags()
+    # Resetting the tx_df flag is required for
+    # continued TX operations when a transmission fails.
+    # clearing the status flags resets the IRQ pin to its inactive state (HIGH)
+    print("\t", end="")
```

Comment:
```suggestion
    print("\t", end="", flush=True)
```
To ensure the `\t` is flushed to stdout before `printStatus()` takes control of stdout.